src/moneypenny/singleton.py

```diff
@@ -1,15 +1,16 @@
 # Stolen from
 # https://stackoverflow.com/questions/6760685/creating-a-singleton-in-python
+from typing import Any, Dict
 
 
 class Singleton(type):
     """Singleton metaclass.  Create a Singleton class with:
-        Class Foo(metaclass=Singleton)
+    Class Foo(metaclass=Singleton)
     """
 
-    _instances = {}
+    _instances: Dict[Any, Any] = {}
 
-    def __call__(cls, *args, **kwargs):
+    def __call__(cls, *args, **kwargs) -> Dict[Any, Any]:  # type: ignore
         if cls not in cls._instances:
             cls._instances[cls] = super(Singleton, cls).__call__(
                 *args, **kwargs
```

Comment:
Do you really need all this? It seems like extra complexity and I'm not sure what it's gaining you.

Reply:
Yeah dump this file

Reply:
removed